src/app/campaign/flight/pingbacks/flight-zone-pingbacks-form.component.ts

```diff
@@ -0,0 +1,109 @@
+import { Component, Input, forwardRef, OnDestroy, ChangeDetectionStrategy } from '@angular/core';
+import { NG_VALUE_ACCESSOR, NG_VALIDATORS, ControlValueAccessor, FormControl, FormArray } from '@angular/forms';
+
+@Component({
+  selector: 'grove-flight-zone-pingbacks',
+  template: `
+    <fieldset>
+      <div *ngFor="let pb of formArray.controls; let i = index" class="inline-fields">
+        <grove-pingback
+          [formControl]="pb"
+          [campaignId]="campaignId"
+          [flightId]="flightId"
+          [podcastId]="podcastId"
+          [creative]="creative"
+        ></grove-pingback>
+        <div *ngIf="formArray.controls.length > 1" class="remove-pingback mat-form-field-wrapper">
+          <button mat-icon-button aria-label="Remove pingback" (click)="onRemovePingback(i)">
+            <mat-icon>delete</mat-icon>
+          </button>
+        </div>
+      </div>
+      <div class="inline-fields">
+        <div class="button-row">
+          <button mat-button color="primary" (click)="onAddPingback()"><mat-icon>add</mat-icon> Add a pingback</button>
+          <a href="https://github.com/PRX/analytics-ingest-lambda#uri-templates" target="_blank" rel="noopener noreferrer">
+            URI syntax help
+          </a>
+        </div>
+      </div>
+    </fieldset>
+  `,
+  styleUrls: ['./flight-zone-pingbacks-form.component.scss'],
+  providers: [
+    { provide: NG_VALUE_ACCESSOR, useExisting: forwardRef(() => FlightZonePingbacksFormComponent), multi: true },
+    { provide: NG_VALIDATORS, useExisting: FlightZonePingbacksFormComponent, multi: true }
```

Comment:
I may have missed this in a past PR... but what exactly do these 2 providers do?  Overriding the accessors/validators on this form-array-ish component?

Reply:
* NG_VALUE_ACCESSOR registers this ControlValueAccessor with angular as a form control.
* NG_VALIDATORS says it has validation